oxenss/rpc/client_rpc_endpoints.h

```diff
@@ -309,6 +309,40 @@ struct delete_msgs final : recursive {
     oxenc::bt_value to_bt() const override;
 };
 
+/// Revokes a Subkey
+///
+/// Takes parameters of:
+/// - pubkey -- the pubkey whose messages shall be deleted, in hex (66) or bytes (33)
+/// - pubkey_ed25519 if provided *and* the pubkey has a type 05 (i.e. Session id) then `pubkey` will
+///   be interpreted as an `x25519` pubkey derived from this given ed25519 pubkey (which must be 64
+///   hex characters or 32 bytes).  *This* pubkey should be used for signing, but must also convert
+///   to the given `pubkey` value (without the `05` prefix).
+/// - revoke_subkey -- the subkey which is to be added to the revocation list, 32 byte sybkey value
+/// - signature -- Ed25519 signature of (subkey); this signs the value constructed
+///   by concatenating "delete" and all `messages` values, using `pubkey` to sign.  Must be base64
+///   encoded for json requests; binary for OMQ requests.
+///
+/// Returns dict of:
+/// - "swarm" dict mapping ed25519 pubkeys (in hex) of swarm members to dict values of:
+///     - "failed" and other failure keys -- see `recursive`.
+///     - "revoke_subkey": list of hashes of messages that were found and deleted, sorted by ascii
+///       value
```

Comment:
We probably don't need actually need to include this in the response, since the person making the request already knows what they requested.